### DISCUSSION TITLE
How to customize `quarkus.oidc.authentication.java-script-auto-redirect=false` browser side header requirements.

### DISCUSSION THREAD

**Discussion by kkazuo** [quarkusio/quarkus#33434] [Q&A] [answered]
Can I customize this `X-Requested-With: JavaScript` browser side cooporation.
https://github.com/quarkusio/quarkus/blob/f6cf2270c54aeb90e21f1b7bf159ce395b88ce0b/docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc#single-page-applications

I use the JS library that add some custom header like `HX-Request: true` automatically.
And so if quarkus-oidc can customize how to detect JS request, it's so nice.

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment, marked as answer]
Hi @kkazuo, I've opened #33436, there are some issues I need to look at first, but we'll take care of it, it makes sense, thanks

**Reply by kkazuo** [reply]
Thank you for consider this.

I say thank you for your great work!
quarkus-oidc helped me a lot already.

**Reply by sberyozkin** [reply]
Thanks @kkazuo for a useful enhancement request, cheers
